x-pack/filebeat/module/cisco/asa/_meta/fields.yml

```diff
@@ -85,3 +85,13 @@
     type: short
     description: >
       ICMP code.
+
+  - name: connection_type
+    type: keyword
+    description: >
+      The VPN connection type
+
+  - name: dap_records
+    type: keyword
+    description: >
+      The assigned DAP records
```

Comment:
Can you add a `default_field: false` entry to both of this new fields? This is necessary to avoid going over the limit of 1024 default_fields in an index template, and we're already dangerously close.